vars.py

```diff
@@ -58,7 +58,7 @@ def set_start_vars(self):
         # === Vars used by the TAS, no pre-set values ===
         # ----Cutscene remover, TAS will set this value on New Game
         # ----If loading to a save file without CSR, may need to change this.
-        self.csr_value = True
+        self.csr_value = False
```

Comment:
Did you mean to commit this change? I feel like not.

Reply:
Reverted, thanks for catching this.